theme/layouts/default.ejs

```diff
@@ -6,10 +6,11 @@
   <body>
     <header>
       <div>
-        <h1><%= page.header %></h1>
-      </div>
-    </header>
-    <% include _navigation %>
+        <h1><a href="#"><%= page.header %></a></h1>
+        <h3><%= page.slogan %></h3>
```

Comment:
Suggestion: Let's define those variables in `config.js` and use `<%= site.title %>` and `<%= site.subtitle %>` instead.

Reply:
OK